doc/src/man/litani-dump-run.scdoc

```diff
@@ -62,6 +63,8 @@ _"complete"_ field set to *true*.
 	every second up to the limit given in this flag. If _N_ is *0*, this program
 	will retry indefinitely---this is not recommended.
 
+*--out-file* _F_
+	Output file to print run.json in.
```

Comment:
I would say something like `Print the run to _F_ instead of stdout`"... I might actually open a separate PR to fix the documentation above, it could be cleaner.

Could you also fix the documentation at the top of the page,

> This program prints the _run.json_ file for an in-progress Litani run to stdout or to the specified output file.

Reply:
Resolved